### ISSUE TITLE
Unsuspending a server causes a connection error in the server list until Wings is restarted

### ISSUE THREAD

**Issue by BolverBlitz** [pterodactyl/panel#3639]
### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.

### Current Behavior

Shows "Connection Error", but server can be used normals.
![grafik](https://user-images.githubusercontent.com/35345288/134425312-a17050ad-5f55-4847-918d-2c480a8b6514.png)

Normal Behavior sometimes after 30-60s, a wings restart on that node will fix it every time.

### Expected Behavior

Should show CPU/RAM/Disk usage, not an error

### Steps to Reproduce

Suspend Server
Look at server while suspended on the server list
Un-Suspend Server
See error message.

Tested on 2 seperate Hosts with diffrent servers/network/hardware.
Tested 7 Times with 100% reproduce rate.

### Panel Version

1.6.2

### Wings Version

1.5.1

### Error Logs

```bash
Does not log an Error. (Tested 23:30) Last error in log 16:48
https://bin.ptdl.co/nwggh/

The error on 16:48 was because i forgot to update wings (Wings 1.4 with Panel 1.6.2)
The reported error happens on all my nodes, not just the one i forgot to update wings before.
```

**Comment by b-kneisel** [comment]
I can confirm it. The same problem also happens to me.

**Comment by miniSpoXster777** [comment]
I can confirm. Same problem

**Comment by b-kneisel** [comment]
Any news regarding this bug?

**Comment by trenutoo** [comment]
There are no news if nothing is posted and bug report is still open.

**Comment by matthewpi** [comment]
TL;DR wings receives the wrong status when suspending a server and unsuspending so it thinks the server is suspended when unsuspended and unsuspended when suspended.

**Comment by anondev-sudo** [comment]
I can confirm this, when I suspend running server, Discord bot is still online and can be used normal.

Server should be killed when it's running when you suspend it.

**Comment by BolverBlitz** [comment]
@anondev-sudo Then you should update to 1.6.3: https://github.com/pterodactyl/panel/blob/develop/CHANGELOG.md#v163
